packages/cli/test/commands/db.test.ts

```diff
@@ -13,14 +13,15 @@ jest.doMock('cross-spawn', () => ({spawn}))
 import DbCmd from '../../src/commands/db'
 
 const schemaArg = `--schema=${path.join(process.cwd(), 'db', 'schema.prisma')}`
+const prismaBin = path.join(process.cwd(), 'node_modules/.bin/prisma')
```

Comment:
Should we be using `resolveCwd` here at all?

Reply:
ah just noticed this was in a test - all good.

Reply:
Good call. I just pushed a commit that changes this to use resloveBinAsync from the server package.